src/Console/Migrate.php

```diff
@@ -25,56 +29,201 @@ class Migrate extends Command
     {
         $this->validateOptions();
         $this->resolveBuilders();
+        $this->resolveOptions();
         $this->cleanTargetDatabase();
         $this->runMigrations();
 
         $this->disableForeign();
         $this->runTransfer();
         $this->enableForeign();
+
+       $this->showStatus();
+    }
+
+
+    protected function showStatus(): void
+    {
+        $this->displayMessage('Migrated Tables', $this->migrated);
+        $this->displayMessage('Excluded Tables', $this->excluded);
+        $this->displayMessage('Tables does not exist in source connection', $this->tables_not_exists);
+    }
+    protected function displayMessage(string $message, array $context = []): void
+    {
+        $this->info($message);
+
+        if ($context){
+            $this->info(implode(',', $context));
+        }
     }
 
     protected function runTransfer(): void
     {
-        $this->info('Transferring data...');
+        $this->displayMessage('Transferring data...' . PHP_EOL);
 
         $this->withProgressBar($this->tables(), function (string $table) {
+            if (in_array($table, $this->excludes)) {
+                $this->excluded[] = $table;
+                return;
+            }
+
+            if ($this->doesntHasTable($this->source(), $table)) {
+                $this->tables_not_exists[] = $table;
+                return;
+            }
+
+            $this->truncateTable($table);
             $this->migrateTable($table, $this->source->getPrimaryKey($table));
         });
+
+        $this->displayMessage(PHP_EOL);
+    }
+
+    protected function truncateTable(string $table): void
+    {
+        if ($this->truncate) {
+            $this->builder($this->target(), $table)->truncate();
+        }
     }
 
     protected function migrateTable(string $table, string $column): void
     {
+        Log::info('Transferring data from: ' . $table);
+
         $this->builder($this->source(), $table)
+            ->when(
+                $this->isSkippable($table, $column),
+                function ($query) use ($table, $column) {
+                    $lastRecord = $this->builder($this->target(), $table)->max($column);
+
+                    Log::info('last record: ' . $lastRecord);
+
+                    return $query->where($column, '>', $lastRecord);
+                }
+            )
             ->orderBy($column)
             ->chunk(1000, function (Collection $items) use ($table) {
                 $items = Arr::toArray($items);
 
                 $this->builder($this->target(), $table)->insert($items);
             });
+
+        $this->migrated[] = $table;
+    }
+
+    protected function isSkippable(string $table, string $column): bool
+    {
+        return ! $this->truncate && $this->isNumericColumn($table, $column);
+    }
+
+    /**
+     * if primary key is not string then skipping existing records
+     * @var string $table - Name of the table
+     * @var string $column - Name of the column
+     * */
```

Comment:
1. The method is self-documenting;
2. There is no need for this dock block.

Besides:
1. The sentence starts with a small letter;
2. There is no full stop at the end of the sentence;
3. Before listing variables there is no indent in the form of an empty line;
4. The dock block ends incorrectly - `*/` must be specified on a new line along with other asterisks.